webui/react/src/components/ExperimentInfoBox.tsx

```diff
@@ -0,0 +1,76 @@
+import { Button } from 'antd';
+import React from 'react';
+
+import Badge, { BadgeType } from 'components/Badge';
+import Link from 'components/Link';
+import { formatDatetime } from 'utils/date';
+import { humanReadableFloat } from 'utils/string';
+
+import { Checkpoint, CheckpointState, ExperimentDetails } from '../types';
+
+import css from './ExperimentInfoBox.module.scss';
+
+interface Props {
+  experiment: ExperimentDetails;
+}
+
+const pairRow = (label: string, value: React.ReactNode | undefined): React.ReactNode => {
+  return (
+    <tr key={label}>
+      <td className={css.label}>{label}</td>
+      <td>
+        {[ 'string', 'number' ].includes(typeof value) ?
+          <span>{value}</span> :
+          <>
+            { value }
+          </>
+        }
+      </td>
+    </tr>
+  );
+};
+
+const InfoBox: React.FC<Props> = ({ experiment: exp }: Props) => {
+  // CHECK orderfactor
+  const orderFator = exp.config.searcher.smallerIsBetter ? 1 : -1;
+  const sortedValidations = exp.validationHistory
+    .filter(a => a.validationError !== undefined)
+    .sort((a, b) => (a.validationError as number - (b.validationError as number)) * orderFator);
+
+  const bestVal = sortedValidations[0]?.validationError;
+
+  const sortedCheckpoints: Checkpoint[] = exp.trials
+    .filter(trial => trial.bestAvailableCheckpoint
+      && trial.bestAvailableCheckpoint.validationMetric
+      && trial.bestAvailableCheckpoint.state === CheckpointState.Completed)
+    .map(trial => trial.bestAvailableCheckpoint as Checkpoint)
+    .sort((a, b) => (a.validationMetric as number - (b.validationMetric as number)) * orderFator);
+
+  // best available checkpoint.
+  const bestCheckpoint = sortedCheckpoints[0];
+
+  return (
+    <div className={css.base}>
+      <table>
+        <tbody>
```

Comment:
@hamidzr let's talk about this one async. In a nutshell, I generally avoid using `table` and the related components because its behavior historically varies so much across browsers, and looking at our normalizer (@import-normalize), it doesn't seem to reset or normalize anything around `table`.